Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,293 @@
+description: Minerva eliminates the endpoint security gap while empowering companies
+  to embrace technology fearlessly.
+detaileddescription: |-
+  Minerva Labs integration with Demisto.
+  The integration queries the Minerva server for new events, endpoint status, groups and able to create active response such as adding vaccinations and exclusions.
+configuration:
+- display: url
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: user
+  name: user
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: password
+  name: password
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: insecure
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+script:
+  script: |
+    # ''' IMPORTS '''
+    import json
+
+    import requests
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    # ''' GLOBALS '''
+    USERNAME = demisto.params().get('user')
+    PASSWORD = demisto.params().get('password')
+    BASE_URL = demisto.params().get('url')
+    VERIFY_SSL = demisto.params().get('insecure')
+
+
+    # ''' HELPERS '''
+    def get_session():
+        session = requests.Session()
+        session.verify = VERIFY_SSL
+        return session
+
+
+    # ''' FUNCTIONS '''
+    @logger
+    def login():
+        return session.post(BASE_URL + '/api/login',
+                            json={'username': USERNAME,
+                                  'password': PASSWORD})
+
+
+    def logout():
+        session.post(BASE_URL + '/api/login/logout')
+
+
+    def fetch_incidents():
+        try:
+            r_events = session.post(BASE_URL + '/api/events',
+                                    json={"archive": False})
+            if r_events.status_code is not 200:
+                return_error('Error in API call. More information: {}, {}'.format(r_exclusion.status_code, r_exclusion.reason))
```

Comment:
`return_error` should not be used in `fetch_incidents` as it uses `demisto.results` which is not relevant in this case. In this scenario, you can use `raise`.